bundles/org.dataflowanalysis.analysis.converter/src/org/dataflowanalysis/analysis/converter/ProcessDFD.java

```diff
@@ -0,0 +1,166 @@
+			List<WebLabel> labels = new ArrayList<>();
+			for(Label label : node.getProperties()) {
+				String labelTypeId = ((LabelType)label.eContainer()).getId();
+				String labelId = label.getId();
+				labels.add(new WebLabel(labelTypeId,labelId));			
+			}
+			
+			List<Port> ports = new ArrayList<>();
+			
+			for (Pin pin : node.getBehaviour().getInPin()) {
+				ports.add(new Port(null,pin.getId(),"port:dfd-input",new ArrayList<>()));
+			}
+
+			Map<Pin, List<AbstractAssignment>> mapPinToAssignments = mapping(node);
+			
+			for (Pin pin : node.getBehaviour().getOutPin()) {
+				String behaviour=createBehaviourString(mapPinToAssignments.get(pin));
+				ports.add(new Port(behaviour,pin.getId(),"port:dfd-output",new ArrayList<>()));
+			}
+			
+			children.add(new Child(text, labels, ports,id,type,null,null,new ArrayList<>()));
+		}
+		
+		DFD output = new DFD (new Model("graph","root",children),labelTypes);
+		
+		ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
+        objectMapper.enable(SerializationFeature.INDENT_OUTPUT);
+
+        try {
+            // Serialize object to JSON and write to a file
+            objectMapper.writeValue(new File(outFile), output);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+	}
+	public Map<Pin, List<AbstractAssignment>> mapping(Node node) {
+		Map<Pin, List<AbstractAssignment>> mapPinToAssignments = new HashMap<>();
+		
+		for (AbstractAssignment assignment : node.getBehaviour().getAssignment()) {
+			if (mapPinToAssignments.containsKey(assignment.getOutputPin())) {
+				mapPinToAssignments.get(assignment.getOutputPin()).add(assignment);
+			} 
+			else {
+				List<AbstractAssignment> list = new ArrayList<>();
+				list.add(assignment);
+				mapPinToAssignments.put(assignment.getOutputPin(), list);
+			}
+		}
+		return mapPinToAssignments;
+	}
+	
+	public String createBehaviourString (List<AbstractAssignment> abstractAssignments) {
+		StringBuilder builder = new StringBuilder();
+		if(abstractAssignments != null) {
+			for (AbstractAssignment abstractAssignment : abstractAssignments) {
+				if (abstractAssignment instanceof ForwardingAssignment) {
+					for (Pin inPin : abstractAssignment.getInputPins()) {
+						builder.append("forward ").append(mapInputPinToFlowName.get(inPin)).append("\n");
+					}
+				} else {			
+					Assignment assignment = (Assignment) abstractAssignment;
+					String value = getTermValue(assignment.getTerm()) ? "TRUE" : "FALSE";
+					
+					for (Label label : assignment.getOutputLabels()) {
+						try {
+							builder.append("set ").append(((LabelType)label.eContainer()).getEntityName()).append(".").append(label.getEntityName()).append(" = ").append(value).append("\n");
+						} 
+						catch (IllegalArgumentException ex) {
+							System.out.println("Caution!! WebEditor cant handle complex Assignments yet. Only TRUE or NOT(TRUE) supported. Everything else ignored");
```

Comment:
That seems to be not correct, at least I was able to specify this (meaningless, but valid) behavior: `set Sensitivity.Public = TRUE || items.Encryption.Encrypted`

Reply:
Could you elaborate a bit?
Where did you set this behavior, in the WebEditor, in the DFD files, or did you manually pass a string containing this behavior?